src/components/events-box.vue

```diff
@@ -5,7 +5,9 @@
          v-show="index <= (showLimit - 1)"
          @click.stop="eventClick(event)"
     >
-     <div class="event"> {{ event.title }} </div>
+     <div class="event">
+       <slot :item="event.title"></slot>
```

Comment:
@shoomakov I would rename the :item to :title since its more clear on whats available in the slot. Then ill merge 👍